guides/common/modules/proc_exporting-a-content-view-version-incrementally.adoc

```diff
@@ -18,14 +18,18 @@ The example below targets the version `2.0` for export because the version `1.0`
 [options="nowrap" subs="+quotes"]
 ----
 # hammer content-export incremental version \
---content-view="_Content_View_Name_" \
+--content-view="_My_Content_View_Name_" \
 --organization="_My_Organization_" \
---version=2.0 \
---format=syncable
+--version=2.0
 ----
+include::snip_syncable-exports.adoc[]
 . Optional: View the exported content view:
 +
-[options="nowrap" subs="+quotes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# ls -lh /var/lib/pulp/exports/_My_Organization_/view/2.0/2021-02-25T21-45-34-00-00/
+# find /var/lib/pulp/exports/_My_Organization_/view/_My_Content_View_Version_/
```

Comment:
Unable to verify this, but I am guessing this is how the exports are stored? Consider this non-blocking if you are unable to verify as well.
```suggestion
# find /var/lib/pulp/exports/_My_Organization_/_My_Content_View_/_My_Content_View_Version_/
```

Reply:
Tested on Foreman 3.9. The name depends on the "--format" option:

````
# hammer content-export complete repository --name "CentOS Client 8 stream" --organization-id 1 --product "CentOS Client"
[.....................................................................................................................................................................................................] [100%]
Generated /var/lib/pulp/exports/Example/Export-CentOS_Client_8_stream-5/1.0/2024-03-08T09-17-27-01-00/metadata.json

# hammer content-export complete repository --name "CentOS Client 8 stream" --organization-id 1 --product "CentOS Client" --format=syncable
[.....................................................................................................................................................................................................] [100%]
Generated /var/lib/pulp/exports/Example/Export-SYNCABLE-CentOS_Client_8_stream-5/1.0/2024-03-08T09-18-21-01-00/metadata.json
````

Org is called "Example".